i18n/module/i18n_module_en.properties

```diff
@@ -151,4 +151,5 @@ no=No
 
 password_changed_successfully=Password changed successfully
 login_again=You need to login again to continue using the application.
-login=Login
\ No newline at end of file
+login=Login
+not_applicable=N/A
```

Comment:
Should we maybe write this out as `Not applicable` rather than `N/A` since it might be a bit more accessible across cultures and easier to translate?

Reply:
Or maybe `None` or `Unknown`?

Reply:
Good point. I think `None` is good, because this is to explain that the event hasn't taken place yet. I'll amend.

Reply:
In the end I thought `Never` would be an even better description for this.